### PR TITLE
fix(ui): Prevent scrolling top when opening toolbox on mobile

### PR DESCRIPTION
**Pull request by TatianaFomina**
- Fixes issue when page is scrolled top if you open toolbox on mobile
- Also contains small typo fix `'Noting'` -> `'Nothing'`
- Adds popup example page

### CODE REVIEW

**Review by neSpecc — APPROVED** [review]
add a changelog please

File: src/styles/ui.css [hunk 1]
```diff
@@ -128,11 +128,9 @@
...
-.ce-scroll-locked, .ce-scroll-locked > body {
-  height: 100vh;
+.ce-scroll-locked {
   overflow: hidden;
-  /**
-   * Mobile Safari fix
-   */
-  position: relative;
+  top: calc(-1 * var(--window-scroll-offset));
+  position: fixed;
```
Comment (neSpecc) [inline src/styles/ui.css:138]: I'm afraid that the reflow of the whole DOM will cost too much

**Review by neSpecc — CHANGES_REQUESTED** [review]

**Review by neSpecc — APPROVED** [review]

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 ### 2.24.1
 
+- `Fix` — Scrolling issue when opening toolbox on mobile fixed
+- `Fix` — Typo in toolbox empty placeholder fixed
+- `Improvement` — *Dev Example Page* - Add popup example page
+
+### 2.24.1
```
Comment (ilyamore88) [inline docs/CHANGELOG.md:9]: Why are there two same versions?

Comment (TatianaFomina) [reply docs/CHANGELOG.md:9]: fixed

**Review by ilyamore88 — APPROVED** [review]

**Pull request merged**